src/App/Fossa/Analyze/Debug.hs

```diff
@@ -162,16 +221,34 @@ collectSystemInfo = do
       processors
       systemMemory
 
+lookUpReachabilityRawScope :: Scope -> Maybe Aeson.Value
```

Comment:
Unfortunately, there is no easy way to persist record entry in debug bundle at root level without inferring from scope. I do think we should make this easy moving forward, and more type constrained.

Reply:
I don't understand- why not just add it like we do e.g. `bundleOutput`?

Reply:
If we want to pipe it similar to `bundleOutput`, we would have to have Analyze's return value, include record of interest. I found this easier, instead of wiring it across nested function calls.